roles/tests-integration/tests/common/mod.rs

```diff
@@ -0,0 +1,194 @@
+use std::sync::Arc;
+
+use async_std::net::TcpStream;
+use key_utils::{Secp256k1PublicKey, Secp256k1SecretKey};
+
+pub async fn start_job_declarator_server() {
+    let authority_public_key = Secp256k1PublicKey::try_from(
+        "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72".to_string(),
+    )
+    .expect("failed");
+    let authority_secret_key = Secp256k1SecretKey::try_from(
+        "mkDLTBBRxdBv998612qipDYoTK3YUrqLe8uWw7gu3iXbSrn2n".to_string(),
+    )
+    .expect("failed");
+    let cert_validity_sec = 3600;
+    let listen_jd_address = "0.0.0.0:34264".to_string();
+    let core_rpc_url = "http://75.119.150.111".to_string();
+    let core_rpc_port = 48332;
+    let core_rpc_user = "username".to_string();
+    let core_rpc_pass = "password".to_string();
+    let coinbase_outputs = vec![jd_server::CoinbaseOutput::new(
+        "P2WPKH".to_string(),
+        "036adc3bdf21e6f9a0f0fb0066bf517e5b7909ed1563d6958a10993849a7554075".to_string(),
+    )];
+    let config = jd_server::Configuration::new(
+        listen_jd_address,
+        authority_public_key,
+        authority_secret_key,
+        cert_validity_sec,
+        coinbase_outputs,
+        core_rpc_url,
+        core_rpc_port,
+        core_rpc_user,
+        core_rpc_pass,
+        std::time::Duration::from_secs(5),
```

Comment:
nit: make this a variable to clearly show what it's used for without needing to check function signature using a LSP or similar.